src/actions/eventActions.js

```diff
@@ -29,122 +28,120 @@ export const fetchEvents = (filter) => reduxBackedPromise(
         getEventsActionSet(filter.incidentId)
     )
 
-export const postEvent = (incidentId, eventTypeId = 0, data= {}, occurrenceTime = moment()) => reduxBackedPromise(
+export const postEvent = (incidentId, eventTypeId = 0, data = {}, occurrenceTime = moment()) => reduxBackedPromise(
     postEventFetchArgs(incidentId, eventTypeId, data, occurrenceTime),
     postEventActionSet(incidentId),
     'POST'
 )
 
-export const getEventsEndPoint = (incidentId) => (incidentId ? 'incidents/' + incidentId + '/': '') + 'events/'
+export const getEventsEndPoint = (incidentId) => (incidentId ? 'incidents/' + incidentId + '/' : '') + 'events/'
 
 export const getEventsFetchArgs = (filter) => ([
-   getEventsEndPoint(filter.incidentId) + filterActions.serializeFiltersForUrl(filter)
+  getEventsEndPoint(filter.incidentId) + filterActions.serializeFiltersForUrl(filter)
 ])
 
 export const getEventFetchArgs = (incidentId, eventId) => {
-    return [getEventsEndPoint(incidentId) + eventId]
+  return [getEventsEndPoint(incidentId) + eventId]
 }
 
 export const postEventFetchArgs = (incidentId, eventTypeId, data, occurrenceTime) => ([
-    getEventsEndPoint(incidentId),
-    {
-        eventTypeId,
-        occurred: occurrenceTime,
-        eventFired: occurrenceTime,
-        data
-    }
+  getEventsEndPoint(incidentId),
+  {
+    eventTypeId,
+    occurred: occurrenceTime,
+    eventFired: occurrenceTime,
+    data
+  }
 ])
 
 export const getEventActionSet = (incidentId, eventId) => ({
-    try: () => ({
-        type: REQUEST_EVENT,
-        incidentId,
-        id: eventId
-    }),
-
-    succeed: (event) => (dispatch) => {
-        dispatch({
-            type: RECEIVE_EVENT,
-            event,
-            id: eventId
-        })
-
-        dispatch(updatePagination())
-    },
-
-    fail: (failureReason) => ({
-        type: RECEIVE_EVENT_FAILURE,
-        failureReason,
-        incidentId,
-        id: eventId
+  try: () => ({
+    type: REQUEST_EVENT,
+    incidentId,
+    id: eventId
+  }),
+
+  succeed: (event) => (dispatch) => {
+    dispatch({
+      type: RECEIVE_EVENT,
+      event,
```

Comment:
I just realized that 'event' is a reserved word in JavaScript. Do you think this could cause issues during local debugging? (I assume it would be minified away in the dist version).

Reply:
Seems it's not a reserved word, but a global variable in some browsers. May be a good idea to change it.